src/main/webapp/stats.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
I tried to add my own statistic (longest message length), in addition to the one provided in the example. I am able to see the correct value in the /stats page, but for some reason, in the /stats.html page, the value shown is "undefined". Can someone please help me with this? I am unable to figure out the source of the error.

Reply:
Hi Meg, the basic reason is that there is no maxMsgLength field in the stats object. The stats object is an object created from the json response string. In your /stats page, there are two fields in your json string, messageCount and longestMessageLength. However, you are calling maxMsgLength instead of longestMessageLength when you try to create your maxMessageElement. Hence, the json object returns undefined for the unknown maxMsgLength field.